package.json

```diff
@@ -37,6 +37,7 @@
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-json": "^4.0.2",
     "@rollup/plugin-node-resolve": "^7.1.1",
+    "babel-eslint": "^10.1.0",
```

Comment:
This PR looks good to me overall. I'm curious why you had to add this and the other estlint package. Did they catch any existing errors? Babel isn't used to build our main output file anymore for es6/npm users and is now only used for the UMD build. I'm guessing this isn't really needed since we've removed the experimental language features

Reply:
Yes, this isn't needed anymore after merging with the most recent master